### DISCUSSION TITLE
Ads aren't showing.

### DISCUSSION THREAD

**Discussion by talisonmarafiga** [admob-plus/admob-plus#352] [General]
Hi guys.

I’m beginner to development and my ads are not showing.

I ran npx admob-plus doctor and it returned the suggestions, but now I don't know how to resolve them.

![image](https://user-images.githubusercontent.com/51460303/119688983-dd896f00-be1e-11eb-9035-02f12f01bc35.png)

**Comment by VictorNorman** [comment]
Can you post your config.xml file here?

**Reply by talisonmarafiga** [reply]
```
<?xml version='1.0' encoding='utf-8'?>
<widget id="com.rhyno.randomMovie" version="1.0.8" xmlns="http://www.w3.org/ns/widgets" xmlns:cdv="http://cordova.apache.org/ns/1.0">
    <name>Random Movie</name>
    <description>
        Random Movie
    </description>
    <author email="marafiga.design@gmail.com" href="marafiga.design@gmail.com">
        Rhyno
    </author>
    <content src="index.html" />
    <access origin="*" />
    <allow-intent href="http://*/*" />
    <allow-intent href="https://*/*" />
    <allow-intent href="tel:*" />
    <allow-intent href="sms:*" />
    <allow-intent href="mailto:*" />
    <allow-intent href="geo:*" />
    <platform name="android">
        <allow-intent href="market:*" />

        <icon src="res/android/ldpi.png" density="ldpi" />
        <icon src="res/android/mdpi.png" density="mdpi" />
        <icon src="res/android/hdpi.png" density="hdpi" />
        <icon src="res/android/xhdpi.png" density="xhdpi" />
        <icon src="res/android/xhdpi.png" density="xxhdpi" />
        <icon src="res/android/xhdpi.png" density="xxxhdpi" />

        <preference name="AndroidXEnabled" value="true" />
        
    </platform>
    <platform name="ios">
        <allow-intent href="itms:*" />
        <allow-intent href="itms-apps:*" />
    </platform>
</widget>
```

**Reply by VictorNorman** [reply]
This [page](https://admob-plus.github.io/docs/cordova/installation) says you need to add two lines to your config.xml to handle the deployment target and swift version stuff.  

Try that first, I think, and then see if that helps with at least part of the problem.

**Reply by talisonmarafiga** [reply]
I didn't identify which two lines.

That `<preference name="AndroidXEnabled" value="true" />` I've added.

**Reply by VictorNorman** [reply]
`<widget>
    <platform name="ios">
        <preference name="deployment-target" value="12.0" />
        <preference name="SwiftVersion" value="5.3" />
    </platform>
</widget>`

If you click on the iOS tab, you'll see the lines above need to be added.

**Reply by talisonmarafiga** [reply]
Oh, okay. Thanks.
